pkg/component/generic/collection/v0/.compogen/bottom.mdx

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
As mentioned in another PR, instead of adding newlines we should put that in the `compogen` template. This is a workaround and contributors (myself included) don't need to know about this.

Reply:
I remove `-` in compogen when there is extra content